### DISCUSSION TITLE
I was charged twice for the$'64

### DISCUSSION THREAD

**Discussion by Jmcstay79** [community/community#48322] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I can't find a phone number to rectify the situation.  Please let me know if there's a # or email.

Thanks!
Janet

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Jmcstay79 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) I'll be locking this post as the community is unable to assist - best of luck!
